gitgud/levels/rampup/_reversing/setup.spec

```diff
@@ -1 +1,7 @@
-
+1 (master)
+2 : 1 (local)
+3 : 1
+4 : 3
+5 : 3
+6 : 4 5 (remote)
+local
```

Comment:
What do you think about this change:
Let's change local to be master and have nothing refer to commit 1. We can then revert the changes, fast forward to the state of the remote and revert the changes from there. Therefore, the correct sequence of commands is going to be
```
git reset HEAD~
git merge remote
git revert 1..5
```

Reply:
I'd like to hear what you think about this. I'm not sure if it's the best way to do it. It might be more logical to do it the way we currently have it. (Although, I still think we should just make the local branch master)

Reply:
Hmmm, I like it better when it's two separate tasks because it allows you to more clearly contrast the two commands.

Reply:
Okay, that's fine then.